packages/mui-material/src/Drawer/Drawer.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { Drawer, PaperProps } from '@mui/material';
```

Comment:
Same as above. To follow https://mui.com/material-ui/guides/minimizing-bundle-size/#option-one-use-path-imports.

Reply:
fixed here https://github.com/mui/material-ui/pull/35924/commits/469e45131652aede236c61316297922a625eeeb9